navigation/AppNavigator.js

```diff
@@ -8,8 +8,24 @@ import MainTabNavigator from "./MainTabNavigator";
 import SignUp from "../screens/SignUpScreen";
 import SignIn from "../screens/SignInScreen";
 import AuthLoadingScreen from "../screens/AuthLoadingScreen";
+import TutorialScreen1 from '../screens/Tutorials/TutorialScreen1';
+import TutorialScreen2 from '../screens/Tutorials/TutorialScreen2';
+import TutorialScreen3 from '../screens/Tutorials/TutorialScreen3';
+import TutorialScreen4 from '../screens/Tutorials/TutorialScreen4';
 
 export const SignedOut = createStackNavigator({
+  Tutorial: {
+    screen: TutorialScreen1
+  },
```

Comment:
In the line following each of these screens, can we add a line that contains:
`navigationOptions: { headerLeft: <></> }`?

This will remove the back arrows from the top of the screen, which I assume isn't wanted in the deisgn
![image](https://user-images.githubusercontent.com/23728476/70364612-13f5b780-1842-11ea-9d4c-fb1cfb55af82.png)